Aufg6/WieseIdee.ts

```diff
@@ -2,23 +2,23 @@ namespace Aufg6_Wiese {
 
     window.addEventListener("load", init);
 
-    let inhalt: CanvasRenderingContext2D;
+    let inhalt: CanvasRenderingContext2D; <-- Besser englische Bezeichnungen verwenden
 
     interface Bee {
 
         x: number;
         y: number;
-        g: number;
+        g: number;  <-- welche Bedeutung soll g haben? x und y ist gut vorstellbar, aber statt g sollt ein sprechender Name stehen

```

Comment:
Jetzt weiß ich das, aber die nächste Person, die den Code liest, steht wieder vor der gleichen Frage
-> daher umbenennen, z.B. in size !!